### PR TITLE
Remove user agent code

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Reviewer:**

## Description:

Instead of https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/569/files and see: https://app.asana.com/0/312629933896096/1200163500750030

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,10 +92,8 @@
         "tabular-json": "0.1.3",
         "tldts": "^5.6.2",
         "underscore": "1.8.3",
-        "useragent": "^2.3.0",
         "web-ext": "^2.7.0",
-        "xvfb": "0.4.0",
-        "yamlparser": "0.0.2"
```
Comment (kdzwinel) [inline package.json:98]: I don't see this being imported anywhere in the removed code - this was a dead dependency?

Comment (jonathanKingston) [reply package.json:98]: useragent uses it internally and the Grunt build process complained if it wasn't included. (kinda the issue at hand here, we got require and yamlparser included into useragent when we were not ever wanting them)

**Review by Charlie-belmer — CHANGES_REQUESTED** [review]
There are a couple of UA related things commented out in the fingerprinting script - assigning values and the getAppVersionValue function. We should probably remove those as well here. Otherwise, looks pretty good to me!

**Review by Charlie-belmer — APPROVED** [review]
Perfect, thanks!

**Pull request merged**